__tests__/bookings.ts

```diff
@@ -1,4 +1,5 @@
 import * as env from "dotenv";
+import gql from "graphql-tag";
```

Comment:
does apollo server not have `gql`?

Reply:
`apollo-server-express` should have it but I don't feel like trusting that being the most stable. I think `graphql-tag` would be the latest and most suitable for testing

Reply:
Sure!